js/modules/export-data.src.js

```diff
@@ -234,6 +234,22 @@ Highcharts.setOptions({
          * @requires modules/export-data
          */
         downloadXLS: 'Download XLS',
+        /**
+         * The text for exported table.
+         *
+         * @since    8.0.4
+         * @requires modules/export-data
+         */
+        exportData: {
+            /**
+             * The category column title.
+             */
+            categoryColumnTitle: 'Category',
+            /**
+             * The category column title when axis type set to "datetime".
+             */
+            categoryColumnTitleDT: 'DateTime'
```

Comment:
DT is not intuitive. 

I would consider to shorter name of param ie: `categoryDatetimeHeader`.